app/src/common/utils/index.js

```diff
@@ -40,3 +40,6 @@ export {
   daysBetween,
 } from './timeDateUtils';
 export { connectRouter } from './connectRouter';
+export { getRolesMap } from './getRolesMap';
+export { getProjectRolesOptions } from './getProjectRolesOptions';
+export { canSelectRole } from './canSelectRole';
```

Comment:
If these methods, finally, are common, I think its a good idea to store them in general rolesUtils.js file